docs/references/ingress-messages.mdx

```diff
@@ -0,0 +1,82 @@
+---
+keywords: [intermediate, reference, security, ingress message, query call, update call, error handling, idempotency]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Ingress messages on ICP
+
+<MarkdownChipRow labels={["Intermediate","Reference", "Security"]} />
+
+## Overview
+
+Ingress messages are calls to ICP canisters that are sent by external entities, such as users, usually through an agent like [agent-js](https://github.com/dfinity/agent-js/tree/main) that runs within a web application, or [agent-rs](https://github.com/dfinity/agent-rs/tree/main) in a Rust application. See also the introductory [call overview](/docs/current/developer-docs/smart-contracts/call/overview). 
```

Comment:
```suggestion
Ingress messages are calls to ICP canisters that are sent by external entities, such as users, usually through an [agent](https://internetcomputer.org/docs/current/developer-docs/developer-tools/dev-tools-overview/#agents) like [agent-js](https://github.com/dfinity/agent-js/tree/main) that runs within a web application, or [agent-rs](https://github.com/dfinity/agent-rs/tree/main) in a Rust application. See also the introductory [call overview](/docs/current/developer-docs/smart-contracts/call/overview). 
```
I linked to the agent definition elsewhere; @jessiemongeon1 , is there perhaps a more robust way to do this with mdx/docusaurus (that would detect if the link breaks)?

Reply:
@oggy-dfin No, there isn't any better way to link to it that would automatically detect if the link breaks. 

We've implemented a few different secondary workflows to catch broken links (hopefully) more often. These are:
- A sitemap analysis script that checks for broken links, and I run it periodically to catch any links that may have been changed. 
- Visitor traffic statistics that send me a weekly report on visitors that received a 404 page and what URL led them to a 404.

Reply:
Thanks, good point, added the link as suggested!